pkg/vendir/config/config.go

```diff
@@ -2,12 +2,10 @@ package config
 
 import (
 	"fmt"
-	"path/filepath"
-	"strings"
-
 	"github.com/ghodss/yaml"
 	semver "github.com/hashicorp/go-version"
 	"github.com/k14s/vendir/pkg/vendir/version"
+	"path/filepath"
```

Comment:
style convention: std lib imports go first with a newline separator for external libs.

Reply:
I just ran Go format the code, but can revert it back

Reply:
updated